### ISSUE TITLE
Adding portrait mode for Camera

### ISSUE THREAD

**Issue by jamesleetu** [flutter/flutter#113670]
## Use case

I need to get depth map from Images that taken from app's camera.
There is no way to force user to use only portrait mode in flutter.
May I request a feature that forcing users to only use portrait mode in camera?
Thank you!

## Proposal

I need MP Image3 metadata for iPhone, and DepthMapData metadata for Samsung mobile phones.
Got no idea for Hwawei, Xiaomi and Oppo.
I need user to take picture from our app and take photo with only portrait mode.
Thank you!

**Comment by jamesleetu** [comment]
@darshankawar 

It's not about portrait and landscape.
![스크린샷 2022-10-20 오후 4 30 20](https://user-images.githubusercontent.com/67309824/196884569-34e4fc1d-5e20-4f3c-8237-01b15a3ca5b2.png)

The portrait mode that I mentioned above is about camera mode

**Comment by jamesleetu** [comment]
@stuartmorgan 
When would be added?

I would like to use new feature as soon as possible :)

Thanks for your work!

**Comment by stuartmorgan** [comment]
Please see https://github.com/flutter/flutter/wiki/Issue-hygiene#when-will-my-bug-be-fixed

**Comment by jamesleetu** [comment]
@stuartmorgan 

Sorry about that
Didn't notice the wiki and again sorry for being rude

Though Thanks for all of your works :)

**Comment by csarigumba** [comment]
@jamesleetu 
Did you figure out how to get the depth data of the image? or using a portrait mode?

@stuartmorgan 
Is there a way to retrieve the depth data of an image in flutter?

**Comment by jamesleetu** [comment]
@csarigumba 
Yes I did figure it out. I used native code opening method channel to get the depth data from image. There is no way to export it only by using Flutter.

**Comment by stuartmorgan** [comment]
Let's please keep discussion focused on this feature request, which is an option for camera to change the capture mode. Reading image data is a completely separate step.
